front-end/src/components/DonorForm/DonorForm.tsx

```diff
@@ -191,10 +190,10 @@ function DonorForm(props: any) {
                 </div>
 
                 <div className ="IsAnon" >
-                <label htmlFor="IsAnon">Donate anonymously?</label>
                 <OverlayTrigger placement="top" overlay={anonTooltip}>
+                <label htmlFor="IsAnon">Donate anonymously?</label>
```

Comment:
should we add an icon so the tooltip is more obvious?
like:
"Donate anonymously? ℹ️"

Reply:
Done 👍